code/drasil-code/lib/Language/Drasil/Choices.hs

```diff
@@ -276,10 +276,10 @@ instance RenderChoices AuxFile where
 
 -- | Gets the file path to a sample input data set from a 'Choices' structure, if 
 -- the user chose to generate a sample input file.
-getSampleData :: Choices -> Maybe FilePath
+getSampleData :: Choices -> FilePath
```

Comment:
See review item below on `sampleFile` for more, but: I think this change is backwards. It uses a 'coding trick' instead of being explicit about the fact that `FilePath` is optional.

Reply:
As I mentioned in the comment above, a way that we could counter this issue would be to create a new choice within the `Choices` area which would take in an integer value which would have a default value of 4. This value essentially would obtain the number of folders that we would have had to go back to in order to get the image. We could then receive any changes to that value for each example and generate the image. That would remove the modifications to the `Maybe Filepath` as our code would no longer be based on the FilePath of the input file.

Let me know if you would like me to modify it such that we create the new `Choice`.

Reply:
Update: The method that I am suggesting in the above comment can be viewed [here](https://github.com/JacquesCarette/Drasil/tree/3418-Fix-Drasil-Logo-Path-on-READMEs-alt) on a different branch.